app/controllers/public/sessions/list.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+import { action, computed  } from '@ember/object';
+
+export default class extends Controller {
+  isTrackVisible = false;
+  @action
+  toggleTrack() {
+    this.toggleProperty('isTrackVisible');
+  }
+
+  @computed('model.event.startsAt', 'model.event.endsAt')
+  get allDates() {
+    const arr = new Array();
+    const difference = (this.model.event.endsAt).diff(this.model.event.startsAt, 'days');
+    for (let i = 0; i <= Math.abs(difference); i++) {
+    	arr.push(moment(this.model.event.startsAt).add(i, 'days').toISOString());
```

Comment:
Codacy found an issue: ['moment' is not defined.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6034570)